src/main/java/at/hannibal2/skyhanni/features/combat/ghostcounter/GhostCounter.kt

```diff
@@ -70,8 +70,8 @@ object GhostCounter {
     val config get() = SkyHanniMod.feature.combat.ghostCounter
     val storage get() = ProfileStorageData.profileSpecific?.ghostCounter
     private var display = emptyList<List<Any>>()
-    var ghostCounterV3File =
-        File("." + File.separator + "config" + File.separator + "ChatTriggers" + File.separator + "modules" + File.separator + "GhostCounterV3" + File.separator + ".persistantData.json")
+    val s = File.separator
+    var ghostCounterV3File = File(""".${s}config${s}ChatTriggers${s}modules${s}GhostCounterV3$s.persistantData.json""")
```

Comment:
This can probably be changed to just using slashes and then wrap the string in `FilenameUtils.separatorsToSystem`